zrml/global-disputes/src/lib.rs

```diff
@@ -0,0 +1,328 @@
+//! # Global disputes
+//!
+//! Manages market disputes and resolutions.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+mod benchmarks;
+mod global_disputes_pallet_api;
+mod mock;
+mod tests;
+pub mod weights;
+
+pub use global_disputes_pallet_api::GlobalDisputesPalletApi;
+pub use pallet::*;
+
+#[frame_support::pallet]
+mod pallet {
+    use crate::{weights::WeightInfoZeitgeist, GlobalDisputesPalletApi};
+    use alloc::vec::Vec;
```

Comment:
better to use `sp_std::vec::Vec`

Reply:
Why? Caio suggested in the past to not use the `std` create and instead use `core` and `alloc`.

Reply:
Mhh, you mean to not use the [`sp_std` library](https://crates.parity.io/sp_std/index.html)? I mean it's sure, that we can't use the `std` (standard library of rust), because it can't be compiled to WASM. I saw this `alloc::vec::Vec` dependency in the prediction markets pallet and thought that this should have a reason, that's why I used it. But on the other hand the `sp_std` library is written by Parity for the use case of blockchains https://docs.substrate.io/main-docs/build/libraries/. So I don't really know in detail, what the difference is, but I would probably lean towards the crate from Parity. Ok, sure, this would result in adding another dependency (`sp_std`).

Reply:
Yes, not to use `sp_std`. You might have pointed out the reason why he suggested that. We already have everything we need in the native crates, so there is no need to add additional dependencies.

Reply:
Corrected in [3e4b556](https://github.com/zeitgeistpm/zeitgeist/pull/682/commits/3e4b556a00cb9db7241fd9c4ac0dbdad3d55ab9b)